.changelog/41678.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:note
+resource/aws_account_region: Change the way to handle region enable/disable when already enabled/disabled in state
```

Comment:
We might want to split this into two distinct enhancement items.

1. Prevent errors when the region is an `ENABLING` or `DISABLING` state during creation.
2. Allow adoption of regions in an `ENABLED` or `DISABLED` state without an explicit import operation.